docs/source/roles/backend.rst

```diff
@@ -0,0 +1,156 @@
+Sensu Go backend role
+=====================
+
+This role installs, configures and starts the Sensu Backend service.
+
+
+Example playbook
+----------------
+
+The most basic
+:download:`backend playbook <../../examples/roles/backend.yaml>` looks like
+this:
+
+.. literalinclude:: ../../examples/roles/backend.yaml
+   :language: yaml
+
+This playbook will install the latest stable version of the Sensu Go backend
+and configure it. We can customize the backend's configuration by adding more
+options to the *backend_config* variable.
+
+
+Backend configuration options
+-----------------------------
+
+The *backend_config* variable can contain any option that is valid for the
+Sensu Go backend version we are installing. All valid options are listed in
+the `official Sensu documentation`_. 
```

Comment:
Same comment as for the agent role - I think it would be beneficial to be more explicit about 1:1 mapping of configuration options. I mean, we kind of already are specific in saying anything that's valid for the `sensu-backend` is also valid here. But I'd like to be very specific about the naming notation.

On second thought, this is already illustrated in the examples for both roles.
Feel free to do as you please :).